fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -2994,6 +2995,38 @@ public void alterMaterializedView(AlterMaterializedViewStatement stmt) throws Dd
         stateMgr.getAlterInstance().processAlterMaterializedView(stmt);
     }
 
+    @Override
+    public void refreshMaterializedView(RefreshMaterializedViewStatement stmt)
```

Comment:
I think we can make the interface 
```suggestion
    public void refreshMaterializedView(String dbName, String mvName)
```
we can handle Statement in Executor. Then this can be called without constructing a statement.